### PR TITLE
docs: Overview and theming of UI Components

### PR DESCRIPTION
**Pull request by szuperaz**
- Update Overview page of UI Components section
- Write about theme customization options in the Theming page
- Remove unused CSS variables from styling package
- Remove unused theming doc from styling package
- Consolidate theme variables

### CODE REVIEW

**Comment by MartinCupela** [comment]
@szuperaz _global-theme-variables.scss are used in demo apps as well as in styling package. We should rather reconcile those with _variables.scss

**Comment by szuperaz** [comment]
> @szuperaz _global-theme-variables.scss are used in demo apps as well as in styling package. We should rather reconcile those with _variables.scss. The same applies to _palette-variables.scss.

uff, now I see it! I randomly searched for variables and didn't spot them being used. Uh, then our variables are all over the place. I'll restore them, but not sure how to reconcile them easily.

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/03-video-theme.mdx [hunk 1]
```diff
@@ -44,3 +44,87 @@ const App = () => {
...
+.str-video {
+  --str-video__primary-color: #6002ee;
+  --str-video__secondary-color: #90ee02;
+  --str-video__text-color1: #282b2b;
+  --str-video__border-radius-circle: 15px;
+  --str-video__icon--download: url('base64 encoded SVG')
+}
+```
+
+### Light and dark themes
```
Comment (szuperaz) [inline packages/react-sdk/docusaurus/docs/React/03-ui-components/03-video-theme.mdx:73]: There are some issues in light mode: https://www.notion.so/stream-wiki/8e8d9743b99f4379b24bd2415e5bfdbd?v=714cda30bb2d433fa7151a33556811c5&p=51a2060477b94d2ebc239aae236fa05a&pm=c - I don't think we will have the capacity to fix these before the beta release

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/01-overview.mdx [hunk 2]
```diff
@@ -7,57 +7,71 @@ description: Overview of the UI components
...
+};
+```
 
 ## Theming with CSS
 
 If you only want to customize the styling of the available components, there is a wide range of CSS variables that you can override.
 With that, you can achieve a significant amount of theming in your video product.
 
-The way this works is that there are various reserved variable names inside of our [styling package](TODO: link to styling package).
 To change them you can simply override them with your custom values.
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/03-ui-components/01-overview.mdx:58]: We may want to mention, that the override has to take place within the scope of `.str-video` class.

Comment (szuperaz) [reply packages/react-sdk/docusaurus/docs/React/03-ui-components/01-overview.mdx:58]: 👍 Updated the docs: https://github.com/GetStream/stream-video-js/pull/546/commits/a27bdac57c907c6fd756a15778a507d03d0174c9

**Review by MartinCupela — APPROVED** [review]

**Review by oliverlaz — APPROVED** [review]

**Review by arnautov-anton — APPROVED** [review]

**Pull request merged**